src/battery/jkbmscan/Stats.cpp

```
@@ -0,0 +1,235 @@
+            //cellno.concat("battery/Cell0");
+            //cellno.concat(str);
+            //cellno.concat("Voltage");
+        }
+        else
+        {
+            cellno="battery/Cell0"+str+"Voltage";
+            //cellno.concat("battery/Cell");
+            //cellno.concat(str);
+            //cellno.concat("Voltage");
+        }
+        MqttSettings.publish(cellno, String(_cellVoltage[i]));
```

Cppcheck (2.20.0)

[error] Array '_cellVoltage[25]' accessed at index 99, which is out of bounds.

(arrayIndexOutOfBounds)